### PR TITLE
Add ra location commands and command handling

### PR DESCRIPTION
**Pull request by arothuis**
Part of [116602125](https://www.pivotaltracker.com/story/show/116602125).

### CODE REVIEW

**Comment by rjkip** [comment]
Continue.

**Comment by arothuis** [comment]
@rjkip?

File: src/Surfnet/Stepup/Tests/Configuration/InstitutionConfigurationTest.php [hunk 1]
```diff
@@ -22,93 +22,16 @@
...
-        $institutionConfiguration->removeRaLocation($raLocationId);
-    }
-
-    /**
-     * @test
-     * @group aggregate
-     */
     public function creating_a_new_institution_configuration_leads_to_an_new_institution_configuration_created_event()
     {
         $institution = new Institution('Test institution');
```
Comment (rjkip) [inline src/Surfnet/Stepup/Tests/Configuration/InstitutionConfigurationTest.php]: I think it's find to remove this last test.

**Comment by rjkip** [comment]
:+1:

**Pull request merged**